test/NITTA/Model/ProcessorUnits/Fram/Tests.hs

```diff
@@ -19,6 +19,7 @@ module NITTA.Model.ProcessorUnits.Fram.Tests (
 
 import Data.Default
 import Data.Text qualified as T
+import NITTA.Intermediate.Functions qualified as F
```

Comment:
Better if you separate such refactoring in different commit. It will simplify review process.